Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM composer:1.7.3 AS composer
+
+COPY composer.json \
+    composer.lock \
+    symfony.lock \
+    ./
+
+RUN composer --no-interaction install --no-dev --ignore-platform-reqs --no-autoloader --no-suggest --prefer-dist
+
+COPY src/ src/
+
+RUN composer --no-interaction dump-autoload --classmap-authoritative
+
+FROM composer AS composer-dev
+
+RUN composer --no-interaction install --ignore-platform-reqs --no-suggest --prefer-dist
+
+FROM php:7.2.11-fpm-alpine as prod
+
+WORKDIR /app
+
+RUN mkdir -p build var && \
+    chown --recursive www-data:www-data var
```

Comment:
files owned by `root` unless differently stated, application running as `www-data`, works for me

Reply:
Maybe should have a `USER www-data` too?

Reply:
I thought that was the default but in fact if `chown` can run here it means it's not, hence a rela need for `USER www-data`